.github/workflows/reusable-build-system-test-react-native.yml

```diff
@@ -50,7 +50,7 @@ jobs:
       - name: Restore CocoaPods cache
         if: ${{ matrix.platform == 'ios' }}
         id: restore-cocoapods-cache
-        uses: actions/cache@0c45773b623bea8c8e75f6c82b208c3cf94ea4f9 # v4.0.2 https://github.com/actions/cache/commit/0c45773b623bea8c8e75f6c82b208c3cf94ea4f9
+        uses: actions/cache@6849a6489940f00c2f30c0fb92c6274307ccb58a # v4.0.2 https://github.com/actions/cache/commit/0c45773b623bea8c8e75f6c82b208c3cf94ea4f9
```

Comment:
Nit: shouldn't this comment say `# v4.1.2`?

Reply:
Yeah that's weird it didn't update, manually updated